index.js

```diff
@@ -20,6 +20,8 @@ module.exports = app => {
 
     const branch = ref.replace(/^refs\/heads\//, '')
 
+    const targetBranch = config['target-branch']

```

Comment:
instead of adding a const I suggest consuming the value from the config object.

Reply:
But then it will need to be consumed on https://github.com/release-drafter/release-drafter/blob/91823a0446bfe714fd7129436d7b09d1af78d056/index.js#L34-L38 and https://github.com/release-drafter/release-drafter/blob/91823a0446bfe714fd7129436d7b09d1af78d056/index.js#L72
Readability will be a little bit worse, I think.